<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>578</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="print">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>390</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>print</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>543</width>
      <height>511</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 500 13pt &quot;Futura&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_back">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>42</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>push_back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="front">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_front">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>push_front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>688</x>
      <y>440</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="empty">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>empty</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pop_front">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>pop_front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pop_back">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>140</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>pop_back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер очереди:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="task">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>221</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Individual task</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
